app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -161,6 +161,11 @@ class DefaultBrowserToolbarController(
                     BrowserFragmentDirections.actionGlobalHome(focusOnAddressBar = true),
                 )
             }
+            is TabCounterMenu.Item.DuplicateTab -> {
+                store.state.selectedTab?.let {
+                    tabsUseCases.duplicateTab(it, selectNewTab = true)
```

Comment:
Did you want to use the new API that you landed over here?

Reply:
Might as well I suppose, though the reason for that API was #27246.